contrib/ohi-release/action.yml

```diff
@@ -0,0 +1,74 @@
+name: OHI release
+description: Wrapper for release toolkit that runs commands needed to release an OHI
+inputs:
+  excluded-dirs:
+    description: 'Exclude commits whose changes only impact files in specified dirs relative to repository root. Defaults to ".github".'
+    required: true
+    default: '.github'
+  fail-if-held:
+    description: 'Fail if the held toggle is active'
+    required: true
+    default: 'true'
+  link-dependencies-enable:
+    description: 'Enables/disables link dependencies step'
+    required: true
+    default: 'false'
```

Comment:
I had to add a toggle to the `link-dependencies` because it fails if the file `.github/rt-dictionary.yaml` does not exist.

I **think** that it is more like in OHIs to not link to any dependency that linking them... That is the reason I default to `false` but we can discuss this during the review.

Reply:
The thing is, we might think if we want the link-dependencies-dictionary to only throw a warning if the file is not found because this command also adds the changelog automatically for git dependencies, and I would like to always have that that feature

Reply:
After taking some time to think about it, from my point of view, the CLI UX is correct: If you set a flag with a path to a file it should fail if it does not exist or you cannot parse it.

As the approach I chose was not correct, I changed the default in [#b75bd12](https://github.com/newrelic/release-toolkit/pull/135/commits/b75bd12b5be2dc02d4f5aae737ed14a9efc90c6f)